label_sync/labels.md

```diff
@@ -184,6 +184,7 @@ larger set of contributors to apply/remove them.
 | <a id="area/control-plane" href="#area/control-plane">`area/control-plane`</a> | Issues or PRs related to control-plane lifecycle management| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="area/health" href="#area/health">`area/health`</a> | Issues or PRs related to health| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="area/machine" href="#area/machine">`area/machine`</a> | Issues or PRs related to machine lifecycle management| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
+| <a id="area/operator" href="#area/operator">`area/operator`</a> | Issues or PRs related to operator| label | |
```

Comment:
```suggestion
| <a id="area/operator" href="#area/operator">`area/operator`</a> | Issues or PRs related to the Cluster API operator| label | |
```

Reply:
These are auto generated

Reply:
Yes, it's generated from YAML, but you can still edit the description there. This description is already under [CAPI section](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-kubernetes-sigscluster-api-for-both-issues-and-prs) so it might not be necessary to specify that operator belongs to CAPI